### ISSUE TITLE
One shortcut for various tools

### ISSUE THREAD

**Issue by F-Burning** [tahoma2d/tahoma2d#694]
**Is your feature request related to a problem? Please describe**

Currently the program has many secondary options, and it can be not only difficult to remember all of them, but also a bit cumbersome to do a single task.

**Describe the solution you'd like**

The ability to jump between tools (or options) with a single key

For example, being able to jump between the different options on the eraser tool only pressing the E key.

**Comment by artisteacher** [comment]
So you can actually do this already - it's just not the default behavior. I kind of wish it was.

In the "Configure Shortcuts" window, under "Tools", set the shortcuts to the ____Tool - Next Mode or ___ Tool - Next Type & Geometric Tool - Next Shape & so on.
<img width="389" alt="Screen Shot 2021-05-10 at 8 02 43 PM" src="https://user-images.githubusercontent.com/21159116/117738720-b9951f00-b1ca-11eb-8807-3dbcd456dbf8.png">
So if I switch "A" to be Animate Tool - Next Mode, the first time I press "A", it switches to the Animate Tool. While the Animate Tool is active, pressing "A" toggles through the different modes.

Similarly, you could set shortcuts for "Mode" and "Type" (under "Tool Modifiers"). Then you would use the same shortcut to toggle through the mode options across all of the tools.

**Comment by F-Burning** [comment]
Yes, but if you want a specific subcategory to work faster, you can't.

For example you can't jump only between two options using one key, because like you said, it toggles through all the modes, but not an specific ones. If I want to use eraser tool and segment eraser tool, I can't use only one key, and if I use the actual method, I need to look constantly to the menu on top to see what tool I choose, instead focus in the canvas and work faster knowing the tools that you are using.

**Comment by artisteacher** [comment]
If you just want to change tools but automatically choose a specific mode/type at the same time, we can do that too. Ie. you can set a shortcut to Eraser Tool - Segment, and it will switch to the eraser tool and set the type to segment  at the same time.

**Comment by F-Burning** [comment]
Yes, I feel this can be useful, because Opentoonz has too much sub-categories inside the own tools, and as you say, this can give people who use keyboard shortcuts a break to work comfortably.
